css-advance/electives.css

```diff
@@ -0,0 +1,108 @@
+.page-content {
+    font-family: 'Times New Roman', Times, serif;
+}
+
+.header {
+    height: 30px;
+    margin-right: 500px;
+    margin-bottom: 10px;
+}
+
+.header>button {
+    float: right;
+    margin-right: 8px;
+    margin-top: 8px;
+}
+
+.all-contents {
+    display: flex;
+    flex-direction: column;
+}
+
+.page-content {
+    display: flex;
+    flex-direction: row;
+}
+
+.elective-wrapper {
+    display: flex;
+    flex-direction: column;
+}
+
+h2 {
+    color: #97bf0e;
+    margin: 0px
+}
+
+.info {
+    font-style: italic;
+}
+
+.button-color {
+    background-color: #97bf0e;
+    color: #ffffff;
+}
+
+.button-shape {

```

Comment:
You may want to change the mouse pointer to hand on hover of the mouse over all the buttons. This way the user will know that the button is clickable. Also: When you click on the button there is this black border appearing around it. It should not be there.